lib/osf-components/addon/components/activity-log/-components/activity-log-display/component.ts

```diff
@@ -0,0 +1,600 @@
+     * @description Abstracted method to build the preprint ahref
+     *
+     * @returns a formatted string
+     */
+    private buildPreprintUrl(): string {
+        if (this.log?.params?.preprint) {
+            return this.buildAHrefElement(`/${this.log.params.preprint}`,
+                this.intl.t('activity-log.defaults.preprint'));
+        }
+        return '';
+    }
+
+    /**
+     * buildPreprintProviderUrl
+     *
+     * @description Abstracted method to build the preprint provider ahref
+     *
+     * @returns a formatted string
+     */
+    private buildPreprintProviderUrl(): string {
+        if (this.log?.params?.preprintProvider) {
+            return this.buildAHrefElement(`/${this.log.params.preprintProvider.url}`,
+                this.log.params.preprintProvider.name);
+        }
+        return '';
+    }
+
+    /**
+     * buildSource
+     *
+     * @description Abstracted method to build the source
+     *
+     * @returns a formatted string
+     */
+    private buildSource(): string {
+        if (this.log?.params?.source) {
+            const source = this.log.params.source;
+            const materialized = this.replaceSlash(source.materialized);
+
+            return this.intl.t('activity-log.defaults.materialized', {
+                materialized,
+                addon: source.addon,
+            });
+        }
+        return this.intl.t('activity-log.defaults.a_name_location');
+    }
+
+    /**
+     * buildTagUrl
+     *
+     * @description Abstracted method to build the tag ahref
+     *
+     * @returns a formatted string
+     */
+    private buildTagUrl(): string {
+        return this.log?.params?.tag ?
+            this.buildAHrefElement(
+                `/search?q=%22${this.log.params.tag}%22`, this.log.params.tag,
+            ) : '';
+    }
+
+    /**
+     * buildTitleNew
+     *
+     * @description Abstracted method to build the new title
+     *
+     * @returns a formatted string
+     */
+    private buildTitleNew(): string {
+        return this.originalNode?.links ?
+            this.buildAHrefElement(
+                `${this.originalNode.links.html}`, this.log.params.titleNew,
+            ) : this.intl.t('activity-log.defaults.a_title');
+    }
+
+    /**
+     * buildUpdatedFields
+     *
+     * @description Abstracted method to build the updated fields
+     *
+     * @returns a formatted string
+     */
+    private buildUpdatedFields(): string {
+        if (this.log?.params?.updatedFields) {
+
+            const nodeCategories =  {
+                analysis: 'Analysis',
+                communication: 'Communication',
+                data: 'Data',
+                hypothesis: 'Hypothesis',
+                instrumentation: 'Instrumentation',
+                'methods and measures': 'Methods and Measures',
+                procedure: 'Procedure',
+                project: 'Project',
+                software: 'Software',
+                other: 'Other',
+            };
```

Comment:
These are untranslated as well. There are translations for this under `node_categories` in the translation file